src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareService.cs

```diff
@@ -231,6 +232,53 @@ await requestContext.SendResult(new ResultStatus()
             }
         }
 
+        /// <summary>
+        /// Handles request for schema compare publish database changes script
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleSchemaComparePublishProjectChangesRequest(SchemaComparePublishProjectChangesParams parameters, RequestContext<SchemaComparePublishProjectResult> requestContext)
+        {
+            SchemaComparePublishProjectChangesOperation operation = null;
+            try
+            {
+                SchemaComparisonResult compareResult = schemaCompareResults.Value[parameters.OperationId];
+                operation = new SchemaComparePublishProjectChangesOperation(parameters, compareResult);
+
+                TaskMetadata metadata = new()
+                {
+                    TaskOperation = operation,
+                    TargetLocation = parameters.TargetProjectPath,
+                    Name = SR.PublishChangesTaskName
+                };
+
+                SqlTask sqlTask = SqlTaskManagerInstance.CreateAndRun<SqlTask>(metadata);
+
+                // very ugly... needs fixing
+                while (operation.PublishResult == null) { };
```

Comment:
Instead:

```
SqlTask sqlTask = SqlTaskManagerInstance.Create<SqlTask>(metadata);
await sqlTask.RunAsync();
```

Reply:
Seems like `SqlTaskManager` doesn't have a `Create` method, as I get the following error when making these changes:
```
'SqlTaskManager' does not contain a definition for 'Create' and no accessible extension method 'Create' accepting a first argument of type 'SqlTaskManager' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
Nevermind, there is a `CreateTask` method, which I am hoping does the same thing. Will test before I resolve this.